test/e2e/specs/help-center/help-center__general-interaction.ts

```diff
@@ -58,11 +60,13 @@ describe( 'Help Center: Interact with Results', function () {
 		} );
 
 		it( 'Click on the first Calypso Link result', async function () {
+			const popupEvent = page.waitForEvent( 'popup' );
 			await supportComponent.clickResultByIndex( 'Calypso Link', 0 );
+			popupPage = await popupEvent;
```

Comment:
This pattern is now suggested by Playwright docs: https://playwright.dev/docs/pages#handling-popups